src/librustc_mir/def_use.rs

```diff
@@ -0,0 +1,170 @@
+// Copyright 2016 The Rust Project Developers. See the COPYRIGHT
```

Comment:
It's a flow-insensitive analysis that creates a map between each local and its mentions. A def-use analysis on non-SSA is typically the flow-sensitive RDA/reaching-definition-analysis.